sdk/appcenter-push/src/main/java/com/microsoft/appcenter/push/Push.java

```diff
@@ -271,16 +271,27 @@ private void enqueuePushInstallationLog(@NonNull String pushToken) {
      */
     @SuppressWarnings("WeakerAccess") /* protected so that Xamarin can use it. */
     protected synchronized void onTokenRefresh(final String pushToken) {
```

Comment:
@guperrot Should it still be `synchronized`?

Reply:
yes for mContext